go.mod

```diff
@@ -16,13 +16,15 @@ require (
 	github.com/ghodss/yaml v1.0.0
 	github.com/go-ini/ini v1.49.0 // indirect
 	github.com/gogo/protobuf v1.3.0 // indirect
+	github.com/golang/groupcache v0.0.0-20200121045136-8c9f03a8e57e // indirect
 	github.com/google/uuid v1.1.1
 	github.com/gophercloud/gophercloud v0.1.0 // indirect
 	github.com/influxdata/influxdb1-client v0.0.0-20190402204710-8ff2fc3824fc
 	github.com/integrii/flaggy v1.2.2
 	github.com/pkg/errors v0.8.1
 	github.com/pkg/sftp v1.10.1 // indirect
-	github.com/sirupsen/logrus v1.4.0
+	github.com/prometheus/common v0.9.1
```

Comment:
Do we need this?

Reply:
Nope, me being unused to go package management and these damm auto updates.
I will remove it in the next commit.